.eslintrc.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
     },
   },
   parserOptions: {
-    project: './tsconfig.json',
+    // project: './tsconfig.json',
+    projectFolderIgnoreList: ['/node_modules/', '/loaders/'],
```

Comment:
Better use eslint's `overrides` to specify different parser for ts and js file. 

https://eslint.org/docs/user-guide/configuring#specifying-processor

Reply:
eg. https://gist.github.com/treecy/437791c02e9edfa2e1006f1da9d34e10

Reply:
the configuration might be different since our eslint version is 6.8.0